### PR TITLE
Added Thai translation

### PR DESCRIPTION
**Pull request by dougmartin**

### CODE REVIEW

**Comment by jsandoe** [comment]
If the minor version number is bumped and this branch is merged to master, I can include this with the CODAP Thai translation in the next build of CODAP.

**Comment by dougmartin** [comment]
@jsandoe I'll get this merged to master by tomorrow.

File: bin/strings-pull-project.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ OUTPUT_DIR=src/code/utils/lang
 # tr: Turkish
 # zh-Hans: Simplified Chinese
 # zh-TW: Traditional Chinese (Taiwan)
-LANGUAGES=("de" "el" "es" "he" "ja" "nb" "nn" "tr" "zh-Hans" "zh-TW")
+LANGUAGES=("de" "el" "es" "he" "ja" "nb" "nn" "tr" "zh-Hans" "zh-TW" "th")
```
Comment (kswenson) [inline bin/strings-pull-project.sh:14]:
```suggestion
LANGUAGES=("de" "el" "es" "he" "ja" "nb" "nn" "th" "tr" "zh-Hans" "zh-TW")
```
For consistency, keep the list alphabetized and add a comment at line 11 above.

File: src/code/utils/translate.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ import ja from './lang/ja'
 import nb from './lang/nb'
 import nn from './lang/nn'
 import tr from './lang/tr'
+import th from './lang/th'
```
Comment (kswenson) [inline src/code/utils/translate.js:20]: It appears that this list was also alphabetized, so bump this up a line.

**Review by kswenson — APPROVED** [review]
👍 Looks good -- just a couple of ordering suggestions.

File: bin/strings-pull-project.sh (hunk 1 above)
Comment (dougmartin) [reply bin/strings-pull-project.sh]: I have re-alphabetized the list

File: src/code/utils/translate.js (hunk 2 above)
Comment (dougmartin) [reply src/code/utils/translate.js:19]: I have bumped it up a line.

**Comment by dougmartin** [comment]
@jsandoe I've merged the Thai translations into the CFM.  Do you also need me to do a tagged release?

**Pull request merged**